packages/teams-js/src/public/authentication.ts

```diff
@@ -428,10 +426,9 @@ export namespace authentication {
    *
    * @param result - Specifies a result for the authentication. If specified, the frame that initiated the authentication pop-up receives
    * this value in its callback or via the `Promise` return value
-   * @param callbackUrl - Specifies the url to redirect back to if the client is Win32 Outlook.
+   * @param _callbackUrl - This parameter is deprecated and unused
    */
-  export function notifyFailure(reason?: string, callbackUrl?: string): void {
-    redirectIfWin32Outlook(callbackUrl, 'reason', reason);
+  export function notifyFailure(reason?: string, _callbackUrl?: string): void {
```

Comment:
This change looks fine, but how would you feel about hiding this function from docs and introducing a new alternate signature that doesn't allow someone to pass in a callback url? Seems like that would lead to less confusion for new people

Reply:
I had considered that when recognizing that I couldn't really "deprecate" the parameter. I'm trying to think about why I didn't take that route. This is not a good reason but I think it was because I didn't know what to call the new functions. 😅